ui/components/component-library/picker-network/picker-network.stories.js

```diff
@@ -38,6 +38,10 @@ export const Label = (args) => (
     <PickerNetwork {...args} label="Arbitrum One" />
     <PickerNetwork {...args} label="Polygon Mainnet" />
     <PickerNetwork {...args} label="Optimism" />
+    <PickerNetwork
+      {...args}
+      label="BNB Smart Chain (previously Binance Smart Chain Mainnet)"
```

Comment:
Can we add a section to the README.mdx docs about max-width and ellipsis?
  ```suggestion
      label="BNB Smart Chain (previously Binance Smart Chain Mainnet)"
      style={{ maxWidth: '200px' }}
```

Reply:
Good shout.